### PR TITLE
[REF]: Scan button moved to tabs

### PR DESCRIPTION
**Pull request by rastajpa**

### CODE REVIEW

File: src/pages/wallets/wallets.html [hunk 1]
```diff
@@ -126,8 +126,4 @@
     <div class="margin-helper"></div>
```
Comment (cmgustavo) [inline src/pages/wallets/wallets.html:126]: Maybe `margin-helper` is not necessary since there isn't float scan button

**Comment by cmgustavo** [comment]
If cancel scanned address, got this error: 
```
ERROR Error: Uncaught (in promise): invalid views to insert
```

**Comment by gabrielbazan7** [comment]
ACK !

**Comment by cmgustavo** [comment]
> "Add as a contact" doesn't copy the address to field "Address"

This still happens in the last commit

**Comment by rastajpa** [comment]
> If cancel scanned address, got this error:
> 
> ```
> ERROR Error: Uncaught (in promise): invalid views to insert
> ```

it doesn't work in the current version. But I will fix it

**Comment by rastajpa** [comment]
> > "Add as a contact" doesn't copy the address to field "Address"
> 
> This still happens in the last commit

it doesn't work in the current version. But I will fix it

**Comment by rastajpa** [comment]
> > "Add as a contact" doesn't copy the address to field "Address"
> 
> This still happens in the last commit

Fixed

**Comment by cmgustavo** [comment]
Nice catch! But the test needs a review too :(

**Comment by rastajpa** [comment]
> Nice catch! But the test needs a review too :(

**Comment by rastajpa** [comment]
Thanks for the review @cmgustavo

**Pull request merged**